dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -51,48 +51,53 @@
   // TODO(suztomo): Add option to specify entry point classes
 
   /**
-   * Returns Maven coordinates for a BOM if specified; otherwise null. Example value: {@code
+   * Returns the Maven coordinates for a BOM if specified; otherwise null. Example value: {@code
    * com.google.cloud:cloud-oss-bom:pom:1.0.0-SNAPSHOT}
    */
   @Nullable
-  abstract String getBomCoordinates();
+  abstract String getBom();
 
   /**
-   * Returns list of coordinates of Maven artifacts if specified; otherwise an empty list. Example
-   * element: {@code com.google.cloud:google-cloud-bigtable:0.66.0-alpha}
+   * Returns list of the coordinates of (non-BOM) Maven artifacts if specified; otherwise an empty
+   * list. Example element: {@code com.google.cloud:google-cloud-bigtable:0.66.0-alpha}
    */
-  abstract ImmutableList<String> getMavenCoordinatesList();
+  abstract ImmutableList<String> getArtifacts();
 
   /**
    * Returns absolute paths for jar files in the filesystem if specified; otherwise an empty list.
    */
-  abstract ImmutableList<Path> getJarFileList();
+  abstract ImmutableList<Path> getJarFiles();
 
   /**
    * Returns {@code true} if only reachable linkage errors should be reported.
    */
   abstract boolean isReportOnlyReachable();
 
   static Builder builder() {
-    return new AutoValue_StaticLinkageCheckOption.Builder();
+    return new AutoValue_StaticLinkageCheckOption.Builder()
+        .setArtifacts(ImmutableList.of())
+        .setJarFiles(ImmutableList.of());
   }
 
   @AutoValue.Builder
   abstract static class Builder {
-    abstract Builder setBomCoordinates(@Nullable String coordinates);
-    abstract Builder setMavenCoordinatesList(List<String> coordinates);
-    abstract Builder setJarFileList(List<Path> paths);
+    abstract Builder setBom(@Nullable String coordinates);
```

Comment:
Do we need the setter to take null? Or should we just not call it if there's no BOM?

Reply:
No it was unnecessary. (I thought it's convention in AutoValue but it's not)

Reply:
They mean two different things.

1.  `@Nullable Foo getFoo()` on the value type means that the property may be `null`.
2.  `Builder foo(@Nullable foo);` on the builder type means that it's okay to pass `null` to the builder.

(2) doesn't make sense unless (1) is true, but even if `foo` is nullable, you could say that it's an error to _set it to `null`_. Instead, just don't call the method if you want it to be `null`.